app/src/main/java/eu/kanade/tachiyomi/ui/manga/chapter/ChaptersSortBottomSheet.kt

```diff
@@ -148,6 +152,31 @@ class ChaptersSortBottomSheet(controller: MangaDetailsController) :
             binding.chapterFilterLayout.setAsDefaultFilter.isInvisible = true
             binding.chapterFilterLayout.resetAsDefaultFilter.isInvisible = true
         }
+        binding.filterGroupsButton.setOnClickListener {
```

Comment:
currently shows nothing if theres no scanlators, and you can exit the dialog by hitting reset to all. Not sure what kind of logic would work. I think sy just shows a check with nothing next to it and you press okay/reset to all too

Reply:
we should add logic before this on whether to show the button or not based on if `presenter.allChapterScanlators.isEmpty()`

Reply:
makes sense, I did `IsNotEmpty()` though and just made it invisible, is there a way to hide it completely and not mess with the xml? I couldn't get it to work as when its invisible, theres a gap between the hide titles and the other filters but when I move it to the bottom, its in the navbar and is essentially unclickable

Reply:
better to do `binding.filterGroupsButton.isVisible = presenter.allChapterScanlators.isNotEmpty()`

Reply:
that would do it if you wanted to keep it there o7